src/dbt_client/exception.ts

```diff
@@ -0,0 +1,40 @@
+import { PythonException } from "python-bridge";
+
+export class CustomException {
+  constructor(
+    public name: string,
+    public message: string,
+    public error?: Error,
+    public extra?: any,
```

Comment:
better to avoid any and use unknown. More details [here](https://javascript.plainenglish.io/what-is-the-unknown-type-in-typescript-5ef6c5333b81#:~:text=In%20Typescript%2C%20any%20value%20can%20be%20assigned%20to%20the%20unknown,without%20first%20asserting%20or%20restricting%20it%20to%20a%20more%20precise%20type.)